### ISSUE TITLE
VGA text output via HDMI?

### ISSUE THREAD

**Issue by techsurrusr** [OLIMEX/VGA2HDMI#1]
The standard text mode for VGA DOS was 80 by 25 characters, which translated to a resolution of 720 by 400 (characters of 9 by 16 pixels), does the VGA2HDMI support this and hopefully then also that it will output BIOS splash screen as well?

**Comment by 8192K** [comment]
Let's give this question a push!

**Comment by DanKoloff** [comment]
The resolutions are not listed as supported by the CV8986 chip. However, I believe it should work, but if you have ideas how can I test it I can do empirical testing.

The PCs with VGA that we have here doesn't seem to have option for low resolutions like 320x240 or 320x200 or 720x400 output. So I did some empirical testing with one of our boards (namely AgonLight2) and it appears VGA2HDMI works fine at 320x240.

Can't say about 320x200 since AgonLight2 firmware for that resolution is set to work at 70Hz or 75Hz and and I don't have display for over 60Hz. Same thing for 720x400 since the firmware of AgonLight2 has no such VGA output resolution configuration in the code. The resolutions I can test with AgonLight2 can be seen as "case" under "int8_t change_mode(uint8_t mode)" around line 525:

https://github.com/OLIMEX/AgonLight2/blob/main/SOFTWARE/2-AgonLight2-VDP-104/source/video/graphics.h

**Comment by 8192K** [comment]
those DOS text mode resolutions usually all came in 70Hz. If you have a PC with VGA output, the BIOS splash, the POST screen as well as anything that's not at least 640x480 should be a text mode resolution. If you can see converted output there, the adapter should work.

**Comment by DanKoloff** [comment]
Well it shows nothing I tested with two older PCs and two different TVs with HDMI. Video output comes at GRUB loader selection and the OS loading. No BIOS logo shown, and if I disable logo in BIOS neither do I see see POST messages. Nothing comes before GRUB if I use VGA2HDMI.

**Comment by 8192K** [comment]
Alright, then it does not work. Thanks for testing!
If, however, you could make this work, then you'd all of a sudden gain a whole new group of potential customers :) (retro gamers)

**Comment by DanKoloff** [comment]
Yeah, I don't think the chip inside supports it. Tho we have a lot of retro gamers fans already, and we manufacture a number of retro computers, check here: https://www.olimex.com/Products/Retro-Computers/

**Comment by sonic74** [comment]
Maybe the attached monitor just doesn't support >60 Hz
Is LED_VGA1 lit?
